### ISSUE TITLE
Dont use GPU

### ISSUE THREAD

**Issue by jiangzhubo** [scheckmedia/keras-shufflenet#5]
when i do training, it only use 1% GPU, and it says 
Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX

Do you know how to fix it

**Comment by scheckmedia** [comment]
For GPU you have to install the package **tensorflow-gpu** instead of the **tensorflow** package.

**Comment by jiangzhubo** [comment]
ofcoz , i have install tensorflow-gpu 1.2.1 version, and it is fine for other deep learning model

**Comment by scheckmedia** [comment]
Hm ok that's strange. But I have no solution for this at the Moment. When I was running the training the usage was between 50% and 90%. The Bottleneck could be the ChannelShuffle but this is just an assumption. 

Uses  a training with MobileNet the GPU better?

**Comment by jiangzhubo** [comment]
use mobilenet , everything is fine, 99%GPU usage.
i tried to do as other said :+1: 
import os
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2' 

but still doesn't work.
my keras verison is 2.1.5

**Comment by scheckmedia** [comment]
I recently tested the training on my GTX 1080:
![image](https://user-images.githubusercontent.com/1307133/42324363-4e9f24a2-8063-11e8-8621-9849bed4f80f.png)

I'm running: 
* tensorflow-gpu               1.8.0
* Keras                        2.2.0
* Keras-Applications           1.0.2
* Keras-Preprocessing          1.0.1
* CUDA 9
